File: src/main/java/jaysonjson/papierfuchs/data/FuchsSize.java
package jaysonjson.papierfuchs.data;

public class FuchsSize {

    private double x = 0;
    private double y = 0;
    private double z = 0;

    public FuchsSize() {

    }

    public FuchsSize(double x, double y, double z) {
        this.x = x;
        this.y = y;
        this.z = z;
    }

    public double getX() {
        return x;
    }

    public double getY() {
        return y;
    }

    public double getZ() {
        return z;
    }

    public void setX(double x) {
        this.x = x;
    }

    public void setY(double y) {
        this.y = y;
    }

    public void setZ(double z) {
        this.z = z;
    }
}

File: src/main/java/jaysonjson/papierfuchs/fuchs/io/data/FuchsSize.kt
package jaysonjson.papierfuchs.fuchs.io.data

import org.bukkit.util.Vector

class FuchsSize {
    var x = 0.0
    var y = 0.0
    var z = 0.0

    constructor()
    constructor(x: Double, y: Double, z: Double) {
        this.x = x
        this.y = y
        this.z = z
    }

    fun vector(): Vector {
        return Vector(x, y, z)
    }
}